resources/analysisTools/rnaseqworkflow/rnaseq_processing.sh

```diff
@@ -68,12 +68,12 @@ then
 	cd $ALIGNMENT_DIR
```

Comment:
Here was a mix of spaces and tabs for indentation, partially caused by using IntelliJ as IDE with spaces set for indentation, but on a file that s.b. else developed with tabs for indententation. This corrects the mess that was the consequence.

The name of the script is changed. See .xml for the corresponding change in the Roddy config.